src/vcRender.h

```diff
@@ -39,13 +47,16 @@ struct vcRenderPolyInstance
   vcTexture *pDiffuseOverride; // optionally override diffuse texture. Only available on RenderType_Polygon
 
   vcGLStateCullMode cullFace;
-  bool ignoreTint;
 
-  bool transparent; // TODO: turn this into appropriate flags
   vcSceneItem *pSceneItem;
   uint64_t sceneItemInternalId; // 0 is entire model; for most systems this will be +1 compared to internal arrays
+
+  // Helper to determine if flag is set
+  bool HasFlag(const enum RenderFlags flag) { return (renderFlags & flag) == flag; }
```

Comment:
This is inconsistent with the other enum flags which do not have "isbitset" functions